web-app/src/components/aboutPage/OurValuesSection.js

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState, useRef } from "react";
+import styled from "styled-components";
+import colours from "../../styles/Colours";
+import FlashCard from "../FlashCard";
+import LoadingSpinner from "../LoadingSpinner";
+import { getFlashCards } from "../../services/flashCards.service";
+import devices from "../../styles/Devices";
+import { useHistory } from "react-router-dom";
+import NoCardSelectedModal from "../NoCardSelectedModal";
+import { getOurValuesCards } from "../../services/ourValuesCards.service";
+
+const OurValuesSection = (props) => {
+  const [noCardSelectedModalVisible, setNoCardSelectedModalVisible] =
+    useState(false);
+
+  const [loading, isLoading] = useState(true);
+
+  const [cardWrapperHeight, setCardWrapperHeight] = useState("100%");
+  const cardWrapperRef = useRef(null);
+
+  let history = useHistory();
+  const [flashCards, setFlashCards] = useState([]);
```

Comment:
could they be referred to as value cards to that we don't confuse this with the help section?